src/Jackett.Common/Definitions/polishtracker-api.yml

```
@@ -87,8 +91,17 @@ search:
       selector: id
     category:
       selector: category
-    title:
+    _language:
+      selector: language
+    _title_org:
+      selector: name
+    _title_pl:
```

```suggestion
    title_phase1:
      selector: name
    title_multilang:
```
in line with other `multi` replacements